src/components/Results/SearchLinkChips.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <section :class="['mb-1 overflow-hidden',{'chips-container-fixed-height':!isColumn}]">
+    <v-chip-group
+      column
+    >
+      <v-chip
+        v-for="(chip,index) in chips"
+        :key="chip.label+'_'+index"
+        text-color="white"
+        :color="getChipColor(chip)"
+        @click.prevent="updateSearchQuery(chip)"
+      >
+        <KeywordTooltip
+          v-if="chip.type === 'subjects' || chip.type === 'domains' "
+          :keyword="chip"
+        />
+        <div
+          v-else
+          class="ellipse-width-80 text-center"
+        >
+          {{ chip.label }}
+        </div>
+      </v-chip>
+      <v-chip
+        v-if="remainTagCount!==0 && remainTagCount!==1"
+        disabled
+        outlined
+        label
+      >
+        {{ `+${remainTagCount} more tags` }}
+      </v-chip>
+      <v-chip
+        v-else-if="remainTagCount===1"
+        disabled
+        outlined
+        label
+      >
+        {{ `one more tag` }}
+      </v-chip>
+    </v-chip-group>
+  </section>
+</template>
+
+<script>
+import {isEqual} from "lodash";
+import KeywordTooltip from "./KeywordTooltip";
+import recordsCardUtils from "@/utils/recordsCardUtils";
+
+export default {
+  name: "SearchLinkChips",
+  components: { KeywordTooltip },
+  mixins: [recordsCardUtils],
+  props: {
+    record: {
+      default: null,
+      type: Object
+    }
+  },
+  data() {
+    return {
+      chips: [],
+      remainTagCount: 0,
+      isColumn: false,
+      getMaxItemShown: 3
+    }
+  },
+  mounted() {
+    this.setChips(this.record);
```

Comment:
Once this function ```this.setChips(this.record)```. It could be directly used in if-else statement above in the template